### ISSUE TITLE
Firefox and ripples.js

### ISSUE THREAD

**Issue by roeku** [mdbootstrap/mdb-ui-kit#2]
I believe the ripples.js does not work properly in firefox as I cannot get it to ripple on any of the elements that do work with chrome.

**Comment by FezVrasta** [comment]
which version of firefox?

**Comment by FezVrasta** [comment]
Ok found the bug, it is

```
 TypeError: target.matches is not a function                                   ripples.js:16
```

Looking for it on caniuse returns http://caniuse.com/#feat=matchesselector

So for old versions of Firefox I should use the prefixed one... will fix it soon.

**Comment by FezVrasta** [comment]
fixed with c19314f14eca06b83b9fdc0fa335acf05ddd3b91
